app/pages/dashboard.tsx

```diff
@@ -82,15 +83,24 @@ const Dashboard = ({
       },
     });
   };
+  
+  const openIntakeBanner = useFeature('open_intake_alert').value || {};
+  const closedIntakeBanner = useFeature('closed_intake_alert').value || {}; 
 
   return (
     <Layout session={session} title="Connecting Communities BC">
       <div>
         <section>
-          <h1>Dashboard</h1>
+          {openIntake && (
+            <DynamicAlert dateTimestamp={closeTimestamp} text={openIntakeBanner.text}
+            variant={openIntakeBanner.variant} includeLink={false} displayOpenDate={false} />
+          )}
           {!openIntake && (
-            <IntakeAlert openTimestamp={nextIntake?.openTimestamp} />
+            <DynamicAlert dateTimestamp={nextIntake?.openTimestamp} text={closedIntakeBanner.text}
+            variant={closedIntakeBanner.variant} includeLink 
+            displayOpenDate = {closedIntakeBanner.displayOpenDate}/>        
```

Comment:
This doesn't look properly formatted. Can you use Prettier to format your files? (VSCode can do that on save with the prettier plugin). Alternatively, the pre-commit hook will ensure your files are formatted

Reply:
Can you please provide .prettierrc with your formatting rules?
I tried with 
```{
  "semi": true,
  "tabWidth": 2,
  "printWidth": 100,
  "singleQuote": true,
  "trailingComma": "none",
  "jsxBracketSameLine": true
}
 
but it produced too many changes

Reply:
we have a `.prettierrc` at the root of this repo

Reply:
updated two files only